models/public/mozilla-deepspeech-0.8.2/mds_convert_utils/memmapped_file_system_pb2.py

```diff
@@ -0,0 +1,125 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Why is this here? Isn't it included in TensorFlow?

Reply:
No, it's not included in TensorFlow (at least in v1.15.3).